### DISCUSSION TITLE
MarkdownTextBlock?

### DISCUSSION THREAD

**Discussion by brduffy** [CommunityToolkit/Windows#269] [Q&A] [answered]
I'm not sure if the MarkdownTextBlock is part of the toolkit or not. I can't find it here but I have downloaded it from the nuget package manager (CommunityToolkit.WinUI.UI.Controls.Markdown) and I'm using it in my project. Having an issue and I'm not sure if its something I can ask about here or not. Anyway when I place table markdown in the text value of my markdown control I get this when I try to go to the next line ...

<img width="956" alt="image" src="https://github.com/CommunityToolkit/Windows/assets/467701/93a2018f-3739-4b78-a487-6e1059ab3a71">

The debugger stops the program and gives me the green arrow indicating that "this is the next statement to execute when this thread returns from the current function". And I see the exception that says "index was outside the bounds of the array". I have not come across this issue when using other markdown like lists or bold, underline, etc. Any ideas?

**Comment by michael-hawker** [comment, marked as answer]
As per the [release notes](https://github.com/CommunityToolkit/Windows/releases/tag/v8.0.230907), we did not bring the MarkdownTextBlock forward; there's a replacement being worked on in Labs that's built on top of the Markdig library: https://github.com/CommunityToolkit/Labs-Windows/pull/480. Feel free to try it and provide feedback.
